src/components/Tooltip/Tooltip.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../styles/common';
 
 $offset-from-activator: rem(4px);
-$content-max-height: rem(100px);
+$content-max-height: rem(500px);
```

Comment:
I would even remove all max-height but maybe someone from Polaris should confirm.

Reply:
I agree to that sentiment. I would be sure that removing `max-height` looks as it should in all supported browsers first. Add me as a reviewer and I can help 🎩 on Monday if you need it

Reply:
+1 for that sentiment!

Reply:
I agree! Can you guys help me get the confirmation to do this? I'm not sure who is part of the Polaris core team

Reply:
@kyledurand is so you're good to go ^^

Reply:
I've just tophatted the removal of max-height from the container and content classes in Firefox, Chrome, Safari, mobile Safari, and chromium edge. All look good to me. If you could update this PR I'll approve 👍

Reply:
Awesome! Just updated, thanks.